README.md

```diff
@@ -58,3 +58,23 @@ and are excluded from the default test run. But if needed, you can run them loca
 ```bash
 $ tox -e integration
 ```
+
+### Windows installation
+
+The `poetry install` command may lock up on Windows10 (Python3.7.6) you can get around this with:
+
+    python -m pip install virtualenv
+    python -m virtualenv .venv                   # IMPORTANT: Notice the dot in the name
+    .venv\Scripts\activate
+    pip install pip-tools
+    poetry export --dev --without-hashes -f requirements.txt > requirements.in
+    pip-compile --upgrade --generate-hashes --output-file requirements.txt requirements.in 
```

Comment:
The poetry hashes (if they are exported) are incomplete, so we use `pip-compile` to list all transitive dependencies.